application/src/main/resources/ecc.yml

```diff
@@ -86,6 +86,15 @@ repair:
     time: 7
     unit: days
   ##
+  ##  The unit of time granularity for priority calculation, can be HOURS, MINUTES, or SECONDS.
+  ##  This unit is used in the calculation of priority.
+  ##  Default is HOURS for backward compatibility.
+  ##  Ensure to pause repair operations when changing the granularity to maintain consistency.
```

Comment:
'when' should probably be something as 'prior to', just to be crystal clear of when to do it.